modules/lite/webpack.config.js

```diff
@@ -4,6 +4,7 @@ const webpack = require('webpack');
 
 const PACKAGE_ROOT = resolve(__dirname, '.');
 const ROOT = resolve(PACKAGE_ROOT, '../..');
+const CORE_VERSION = require(resolve(ROOT, 'node_modules/@deck.gl/core/package.json')).version;
```

Comment:
Nit: Maybe add a comment about what is going on here. 

E.g. I assume you have verified that lerna is going to bump `core` version before getting here and so the imported version string is guaranteed to be correct, that could be nice to mention in a comment.

Reply:
Yes, we rebuild the core module and this bundle in `prePublishOnly` which is called after the version bump.